onos-classic/values.yaml

```diff
@@ -54,3 +58,14 @@ ports:
 #     log4j2.appender.console.name = Console
 #     log4j2.appender.console.layout.type = PatternLayout
 #     log4j2.appender.console.layout.pattern = %d{RFC3339} %-5level [%c{1}] %msg%n%throwable
+
+podSecurityContext:
```

Comment:
again might be worth run-in with these two disabled by default @kuujo thoughts ?

Reply:
no because default should be a secure setup and if a user wants to make it insecure he should do it explicitly otherwise we provide a non-secure definition by default